### ISSUE TITLE
[Request] Provide Linux x86 binaries

### ISSUE THREAD

**Issue by alicaccs** [fabiofalci/sconsify#36]
Surely everyone can download the source and build it on their own, but Spotify API key is still required to build and run sconsify. It would be some kind of redundant work for every x86 user to request a key just for building a single copy of sconsify. Providing x86 binaries would be grateful for people who stick with old computers.

**Comment by fabiofalci** [comment]
That sounds a good idea indeed.
However I only have access to x86_64 OSs, assuming you are talking about linux, right? don't know if we can have x86 osx these days.

I'd be more than happy to try cross compilation in that case but it involves native library access,  libspotify: https://developer.spotify.com/technologies/libspotify/
So I'm not sure if that's possible.

**Comment by uyuarasy** [comment]
In case you still don't have access to an x86 PC, could you expand the installation guidance? I'd like to install Sconsify to a Debian Wheezy, but I can't compile from source. The Go 1.6.2 (first from golang package, then from source) and the Glide worked fine, but at the final step I can't build the Sconsify package (the path was fine).

**Comment by fabiofalci** [comment]
The only missing instructions in the readme is the golang version.
You need to build with 1.5 version. 
1.6 won't work as it has changed some pointer rules.

**Comment by uyuarasy** [comment]
Is golang 1.0.2 obsolete? With glide 0.10.2 and the gitcloned sconsify the 'glide install' fetching and setting process was fine but for the 'make build' I get the following error:

`[12:50]user@pc(2):sconsify[0]$ make build
/home/xinticell/gocode/bin/glide -q install
sed -i '$ d' spotify/key.go && cat spotify/spotify_key_array.key >> spotify/key.go \
                && GO15VENDOREXPERIMENT=1 go build -ldflags "\
                 -X main.version=0.3.0-local \
                 -X main.commit=94e18f5 \
                 -X main.buildDate=1463830201 \
                 -X main.spotifyClientId=4e1fa8c468ce42c2a45c7c9e40e6d9d0 \
                 -X main.authRedirectUrl=https://fabiofalci.github.io/sconsify/auth/" \
                 -o bundles/sconsify \
                ; git checkout spotify/key.go
infrastructure/env.go:8:2: import "github.com/fabiofalci/flagrc": cannot find package
spotify/portaudio.go:5:2: import "github.com/gordonklaus/portaudio": cannot find package
sconsify.go:18:2: import "github.com/howeyc/gopass": cannot find package
ui/simple/cui.go:12:2: import "github.com/jroimartin/gocui": cannot find package
infrastructure/env.go:9:2: import "github.com/mitchellh/go-homedir": cannot find package
sconsify/track.go:6:2: import "github.com/op/go-libspotify/spotify": cannot find package
webapi/web_auth.go:8:2: import "github.com/zmb3/spotify": cannot find package
`

When I gitcloned the packages to the $GOPATH/src folder, it was the same. Should I try with 'go get package'? With that there were various errors.

**Comment by fabiofalci** [comment]
It uses golang `vendor` directory which was introduced in 1.5.
So probably yes, 1.0.2 is obsolete.

**Comment by uyuarasy** [comment]
I see. Now I have go1.5.1, the packages above were installed with `go get .. ` then the `make build` command was fine. The `go run sconsify.go` was started the program in text-mode, and it's awesome! 
Thanks! :)

**Comment by fabiofalci** [comment]
Cool stuff.
Probably you are using the master to build, right?

Just a warning that sometimes it freezes and I still don't know why. If you check the git tags you can get the latest release, that should be more stable.

**Comment by uyuarasy** [comment]
Yes, I am used the master as source and the build works flawlessly. Keep up the good work!

**Comment by sauloaugustosilva** [comment]
Hi uyuarasy , can you share your binary for Debian x86 ?

**Comment by uyuarasy** [comment]
Hello @sauloaugustosilva, a few months ago I had to apply major updates on my system, the sconsify no longer works and I'm not tried to fix it. I still have the files, which one are you looking for?

**Comment by sauloaugustosilva** [comment]
Hi @uyuarasy , I am looking for the binaries of sconsify for debian x86  .
